base/common/src/test/java/com/netscape/certsrv/profile/ProfileDataTest.java

```diff
@@ -0,0 +1,65 @@
+package com.netscape.certsrv.profile;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ProfileDataTest {
+
+    private static ProfileData before = new ProfileData();
+    private static List<ProfileInput> inputs = new ArrayList<>();
+    private static ProfileInput input = new ProfileInput();
+
+    @Before
+    public void setUpBefore() {
+        input.setClassId("foo");
+        input.setName("bar");
+        input.setText("lorem");
+
+        inputs.add(input);
```

Comment:
I suppose this code has the same issue: the list of inputs will get longer in subsequent tests.

Reply:
Fixed, thanks!